py/server/deephaven/table.py

```diff
@@ -1293,10 +1293,10 @@ def agg_by(self, aggs: Union[Aggregation, Sequence[Aggregation]], by: Union[str,
                 grouped into a single group of rows before the aggregations are applied to the result, default is None.
             preserve_empty (bool): whether to keep result rows for groups that are initially empty or become empty as
                 a result of updates. Each aggregation operator defines its own value for empty groups. Default is False.
-            initial_groups (Table): a table whose distinct combinations of values for the group-by column name(s)
+            initial_groups (Table): a table whose distinct combinations of values for the group-by column(s)
                 should be used to create an initial set of aggregation groups. All other columns are ignored. This is
-                useful in combination with preserve_empty == True to ensure that particular groups appear in the result
-                table, or with preserve_empty == False to control the encounter order for a collection of groups and
+                useful in combination with preserve_empty=True to ensure that particular groups appear in the result
+                table, or with preserve_empty=False to control the encounter order for a collection of groups and
                 thus their relative order in the result. Changes to this table are not expected or handled; if this
                 table is a refreshing table, only its contents at instantiation time will be used. Default is None,
```

Comment:
Does instantiation mean when `initial_groups` is created or when this method is called?  I'm guessing the latter, but this is not clear.